src/pages/index.js

```diff
@@ -1,15 +1,26 @@
-import "./index.css";
+import "../pages/index.css";
 import logoPath from "../images/logo.svg";
 import pencilPath from "../images/pencil.svg";
 import plusPath from "../images/plus.svg";
 import avatarPath from "../images/avatar.jpg";
+import closeIcon from "../images/close-icon.svg";
 import { enableValidation, resetValidation } from '../scripts/validation.js';
+import Api from "../scripts/Api.js";
+
 
 document.querySelector(".header__logo").src = logoPath;
 document.querySelector(".profile__edit-btn img").src = pencilPath;
 document.querySelector(".profile__add-btn img").src = plusPath;
 document.querySelector(".profile__avatar").src = avatarPath;
```

Comment:
`document.querySelector(".profile__avatar")`

You've already found the avatar in line 66 as `profileAvatar`. Please, move this definition onto line 10, to reuse it here

```js

const profileAvatar = document.querySelector(".profile__avatar");

document.querySelector(".header__logo").src = logoPath;
document.querySelector(".profile__edit-btn img").src = pencilPath;
document.querySelector(".profile__add-btn img").src = plusPath;
profileAvatar.src = avatarPath;
```